fast64_internal/oot/oot_actor.py

```diff
@@ -1,47 +1,410 @@
-
-import math, os, bpy, bmesh, mathutils
-from bpy.utils import register_class, unregister_class
-from io import BytesIO
+import bpy
 
 from ..f3d.f3d_gbi import *
 from .oot_constants import *
 from .oot_utility import *
-
 from ..utility import *
+from .oot_operators import *
 
-class OOT_SearchActorIDEnumOperator(bpy.types.Operator):
-	bl_idname = "object.oot_search_actor_id_enum_operator"
-	bl_label = "Select Actor ID"
-	bl_property = "actorID"
-	bl_options = {'REGISTER', 'UNDO'} 
+# General classes and functions
 
-	actorID : bpy.props.EnumProperty(items = ootEnumActorID, default = "ACTOR_PLAYER")
-	actorUser : bpy.props.StringProperty(default = "Actor")
+class OOT_UpgradeActors(bpy.types.Operator):
+	bl_idname = 'object.oot_upgrade_actors'
+	bl_label = "Upgrade Data Now!"
```

Comment:
`bl_label` should be more (very briefly) descriptive of the operator
Maybe `bl_label = "Upgrade Fast64 OoT actors data"`

and "Upgrade Data Now!" can be used for the ui like `actorIDBox.operator(OOT_UpgradeActors.bl_idname, text="Upgrade Data Now!")`

Also you can test this if you mimic blender handlers not working, which seems to be what happens for me sometimes, for example by removing https://github.com/Fast-64/fast64/blob/347d19e3fbc8449aaba91de337952fe3d738f4b9/__init__.py#L359